homework2/task_2.py

```diff
@@ -0,0 +1,9 @@
+# Напишите программу, которая принимает на вход число N и выдает набор произведений чисел от 1 до N.
+
+num = int(input("Введите число: "))
+
+lst = [1]
+for i in range(1, num + 1):
+    lst.append(i * lst[- 1])
```

Comment:
```suggestion
    lst.append(i * lst[-1])
```
Это унарный минус, он пишется без пробела.